pages/linux/deluser.md

```diff
@@ -0,0 +1,20 @@
+# deluser
+
+> Delete user from system
+> All commands must be executed as root (`sudo [command]`)
+
+- Remove a user:
+
+`deluser {{name}}`
+
+- Remove a user and their home directory:
+
+`deluser --remove-home {{name}}`
+
+- Remove a user and their home, but backup it's files first:
+
+`deluser --backup-to {{path/to/backup}} --remove-home {{name}}`
```

Comment:
I assume this is a directory?
```suggestion
`deluser --backup-to {{path/to/backup_directory}} --remove-home {{username}}`
```

Reply:
Oh, yes, it's a directory where one `.tag.gz` file will be placed

Reply:
Oh it's an archive?  Then I would clarify this in the description.